versioneer.py

```diff
@@ -280,7 +280,7 @@
 try:
     import configparser
 except ImportError:
-    import ConfigParser as configparser
+    import ConfigParser as configparser # type: ignore[no-redef]
```

Comment:
is it ok for us to leave this ignore comment? I wasn't sure how else to avoid mypy's "Name 'configparser' already defined (by an import)" error

Reply:
Yeah, that's fine.